src/cpu.c

```diff
@@ -13,9 +13,25 @@ typedef void (*op_function)(void);
 // Function is used for instruction array initialization, not recognized by compiler
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wunused-function"
+
 static void UNKNOWN_OPCODE(void) {
+#if defined(YOBEMAG_DEBUG)
+
+    static unsigned err_count = 0;
     LOG_ERROR("Unallowed/Unimplemented OP Code: 0x%x", cpu.opcode);
+    if (++err_count > 100) {
+        LOG_FATAL("Exceeded number of unallowed OP codes!");
+        exit(EXIT_FAILURE);
```

Comment:
I think you could use `YOBEMAG_EXIT("...")` here.

Reply:
Forgot that that exists